api_app/management/commands/migrate_secrets.py

```diff
@@ -22,6 +22,13 @@ def _get_env_var(name):
     def _migrate_secrets(cls, plugin_list, plugin_type):
         from django.contrib.auth import get_user_model
 
+        if PluginConfig.objects.filter(plugin_type=plugin_type).exists():
+            print(
+                f"Skipping {plugin_type} secrets migration because "
+                "there are already some secrets in the database."
+            )
+            return
+
         User = get_user_model()
         if not User.objects.filter(is_superuser=True).exists():
             raise Exception("Superuser must exist for secrets migration")
```

Comment:
I think this won't work in a clean environment because the superuser still needs to be created manually right now.

On the other hand, in a clean environment the user won't need to import anything so I think that the flow makes sense like it is. Am I right?

Reply:
Yup. I assumed any working environment would always have a superuser.